src/Models/Address.php

```diff
@@ -253,6 +254,13 @@ public function addressTypes(): BelongsToMany
         return $this->belongsToMany(AddressType::class);
     }
 
+    public function addressTypeOrder(): BelongsToMany
```

Comment:
addressTypeOrder before addressTypes
And shouldn't the relation name be addressTypeOrders() as it returns not a singular order?